.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -0,0 +1,26 @@
+name: Warn on editing requirements
+
+on:
+  pull_request_target:
+    paths:
+      - setup.py
+      - requirements/skinny-requirements.yaml
+
+permissions:
+  pull-requests: write
+
+jobs:
+  warn:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post warning if file is edited
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: "⚠️ Warning: It looks like you're editing requirements for MLflow. Please ensure that this doesn't cause any dependency conflicts with downstream tasks that depend on MLflow (e.g. image building).",
```

Comment:
```suggestion
              body: "⚠️ Warning: It looks like you're editing requirements for MLflow. Please ensure that this doesn't cause any dependency conflicts with downstream tasks that depend on MLflow (e.g. Databricks runtime image building).",
```

Can we mention `Databricks runtime image building`? `image building` sounds a bit ambiguous.